### PR TITLE
[imagemagick6] Update to 6.9.11-29

### PR DESCRIPTION
**Pull request by vincowl**

### CODE REVIEW

File: packages/imagemagick6.rb [hunk 1]
```diff
@@ -63,7 +58,7 @@ def self.build
            --with-perl \
            --with-rsvg \
            --with-x"
-    system 'make'
+    system "make -j#{CREW_NPROC}"
```
Comment (uberhacker) [inline packages/imagemagick6.rb:61]: This is not necessary.  The default value of `-j` is set to this value.  See `lib/package.rb`.

Comment (vincowl) [reply packages/imagemagick6.rb:61]: Fine ! I'll remove it from imagemagick too then.

File: packages/imagemagick6.rb [hunk 2]
```diff
@@ -55,6 +49,7 @@ def self.build
            ./configure \
            --prefix=#{CREW_PREFIX} \
            --libdir=#{CREW_LIB_PREFIX} \
+           --mandir=#{CREW_MAN_PREFIX} \
```
Comment (uberhacker) [inline packages/imagemagick6.rb:52]: Does `CREW_OPTIONS` work here?

Comment (vincowl) [reply packages/imagemagick6.rb:52]: No. It works if you remove `--target` from `#{CREW_OPTIONS}`. Don't know why....

**Pull request merged**